### ISSUE TITLE
Map not showing on entry form

### ISSUE THREAD

**Issue by eldelacajita** [EcosistemaUrbano/local_in#79]
The map doesn't appear when trying to geolocate an entry. 

![image](https://user-images.githubusercontent.com/2880679/43071277-8f92f840-8e72-11e8-8580-e23f60e52031.png)

This error has been seen in at least two different instances of local_in 3.0

**Comment by xomnot** [comment]
This is related with the explorer (I've tested with Firefox and Chromium). It disables the map, because is "mixed content". 
It happens when the main page is "https" but, it seems that the map loads from some "http" and the explorer blocks it as suspicious activity

The addresses that need to be changed in case of https website are on "header.php"

**Comment by eldelacajita** [comment]
Thanks for reporting back! 

So it only happened to you when having `https` enabled on the main page? I think it wasn't the case for me. I'll look into it.

**Comment by xomnot** [comment]
Yes... I used to have the page hosted in another hosting, that was without ssl and HTTP, and there were no problems.

Once migrated the web, to another hosting which was https (the whole web, not only main page) the map disappeared. Blocked by the explorer for "mixed content".

Changing the addresses on header.php which linked to google maps from http to https, solved the problem to me.

**Comment by eldelacajita** [comment]
Great! I'll try it on my setup, and keep this issue open until we commit a permanent fix for this.
